src/main/java/com/softserveinc/ita/pageobjects/modals/AddingSpecialtyModal.java

```diff
@@ -0,0 +1,18 @@
+package com.softserveinc.ita.pageobjects.modals;
+
+import io.qameta.allure.Step;
+
+import static com.codeborne.selenide.Condition.appear;
+import static com.codeborne.selenide.Selenide.$x;
+import static java.time.Duration.ofSeconds;
+
+public class AddingSpecialtyModal extends BaseModal<AddingSpecialtyModal> {
+    @Override
+    @Step("Adding specialty modal: Added new specialty")
+    public void addNewItem() {
+        $x("//button/span[contains(text(),'Підтвердити')]")
```

Comment:
instead of overriding you use common selector for button in BaseModal "//form/button[@type='submit' or ./span[contains(text(),'Додати')] or ./span[contains(text(),'Підтвердити')]]"